website/src/pages/community/promotion.js

```diff
@@ -0,0 +1,211 @@
+    ],
+    permissions: 'Write Permission',
+    timeline: 'After each release (2-3 month intervals)',
+    application: 'Must be nominated by a maintainer, requires majority vote from maintainers',
+    color: '#2196F3',
+  },
+  {
+    role: 'Maintainer',
+    icon: '🛠️',
+    level: 3,
+    requirements: 'Sustained contributions across three consecutive releases',
+    details: [
+      'Demonstrate long-term project commitment',
+      'Lead major feature development in workgroups',
+      'Drive important refactoring initiatives',
+      'Champion critical performance optimizations',
+      'Guide and mentor other contributors',
+      'Shape project direction and roadmap',
+    ],
+    permissions: 'Maintain Permission',
+    timeline: 'After each release (2-3 month intervals)',
+    application: 'Must be nominated by a maintainer, requires unanimous approval from all maintainers',
+    color: '#FF9800',
+  },
+]
+
+function PromotionCard({ rule }) {
+  return (
+    <div className={styles.promotionCard} style={{ borderColor: rule.color }}>
+      <div className={styles.cardHeader}>
+        <span className={styles.roleIcon}>{rule.icon}</span>
+        <h3 className={styles.roleTitle} style={{ color: rule.color }}>{rule.role}</h3>
+        <span className={styles.permissions} style={{ backgroundColor: rule.color }}>
+          {rule.permissions}
+        </span>
+      </div>
+
+      <div className={styles.cardContent}>
+        <div className={styles.requirements}>
+          <h4>📋 Requirements</h4>
+          <p className={styles.mainRequirement}>{rule.requirements}</p>
+          <ul className={styles.detailsList}>
+            {rule.details.map((detail, index) => (
+              <li key={index}>{detail}</li>
+            ))}
+          </ul>
+        </div>
+
+        <div className={styles.timeline}>
+          <h4>⏰ Timeline</h4>
+          <p>{rule.timeline}</p>
+        </div>
+
+        <div className={styles.application}>
+          <h4>📝 How to Apply</h4>
+          <p>{rule.application}</p>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+export default function Promotion() {
+  return (
+    <Layout
+      title="Promotion"
+      description="vLLM Semantic Router Community Promotion Rules"
+    >
+      <div className={styles.container}>
+        <header className={styles.header}>
+          <h1>Community Promotion 🚀</h1>
+          <p className={styles.subtitle}>
+            Contributor advancement rules - Recognizing your contributions and elevating your impact
+          </p>
+        </header>
+
+        <main className={styles.main}>
+          <section className={styles.overview}>
+            <h2>📖 Promotion Overview</h2>
+            <div className={styles.overviewContent}>
+              <div className={styles.overviewCard}>
+                <h3>🎯 Promotion Timing</h3>
+                <p>
+                  Promotions occur after each release, with
+                  <strong>2-3 month</strong>
```

Comment:
```suggestion
                  <strong> 2-3 month</strong>
```
we need to add a space (or &nbsp)
<img width="1349" height="363" alt="image" src="https://github.com/user-attachments/assets/4ec90f08-b98b-406b-8f59-a2246101d83f" />